recipes/fastapi-azure-auth/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fastapi-azure-auth" %}
+{% set version = "3.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 51e514b3a89df2e730929b501237508763a6aa88e4aab0a71665a4d000a7e789
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - cryptography >=38.0.3
+    - fastapi >0.68.0
+    - httpx >0.18.2
+    - pip
+    - poetry
+    - python
+    - python-jose[cryptography] >=3.3.0,<4.0.0
```

Comment:
This syntax for option deps is not supported by conda. IDK what precisely to do here but I think you'll need to ensure that the conda version of this package has the optional stuff included.

Reply:
Thanks, have updated to remove `[cryptography]` here.